pkg/background/update_request_controller.go

```diff
@@ -178,39 +175,30 @@ func (c *Controller) syncUpdateRequest(key string) error {
 	defer func() {
 		logger.V(4).Info("completed sync update request", "key", key, "processingTime", time.Since(startTime).String())
 	}()
-
 	_, urName, err := cache.SplitMetaNamespaceKey(key)
 	if err != nil {
 		return err
 	}
-
-	ur, err := c.urLister.Get(urName)
-	if err != nil {
-		if apierrors.IsNotFound(err) {
-			return nil
+	if ur, err := c.urLister.Get(urName); err != nil {
+		return err
+	} else {
```

Comment:
I usually prefer flat code structure where we return on error and then implement the rest of the logic from a new line. For example:
```
if err != nil {
  return
}

// Do something else
```

Otherwise there will be too many if-else blocks and brackets which makes code not easy to read. This is just my personal preference though :)

Reply:
Agree, I too prefer the flat code structure for better readability.

Reply:
Fixed.